examples/arithmetic/Cargo.toml

```diff
@@ -10,11 +10,8 @@ crate-type = ["cdylib"]
 name = "uniffi_arithmetic"
 
 [dependencies]
-anyhow = "1.0"
-log = "0.4"
-ffi-support = "0.4"
-uniffi = {path = "../../uniffi"}
 uniffi_macros = {path = "../../uniffi_macros"}
+uniffi = {path = "../../uniffi"}
 
 [build-dependencies]
-uniffi = {path = "../../uniffi"}
+uniffi_build = {path = "../../uniffi_build"}
```

Comment:
See how we depend on 1 crate for the bytebuffer traits/methods + tests helpers, and 1 for build time.